config/main.py

```diff
@@ -27,6 +27,9 @@
 SYSLOG_IDENTIFIER = "config"
 VLAN_SUB_INTERFACE_SEPARATOR = '.'
 ASIC_CONF_FILENAME = 'asic.conf'
+PLATFORM_ROOT_PATH = '/usr/share/sonic/device'
```

Comment:
This constant appears to be unused. Remove?

Reply:
Removed.